packages/opentelemetry-node/docs/metrics.mdx

```diff
@@ -20,7 +24,7 @@ node -r @elastic/opentelemetry-node/start.js my-app.js
 You can tune how often metrics data is exported to the endpoint and the max time
 to export data you can use the env vars already defined in [the spec](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#periodic-exporting-metricreader).
 
-## Host metrics
+## Host metrics
```

Comment:
nit: 🤔 What is that change?

Reply:
I don't know. This heading wasn't rendering correctly and I couldn't figure out why. I eventually deleted it and replaced it. It appears there was a blank/empty non-space character here before. I'm not sure what though.

Reply:
Interesting. Looks like it was a Unicode "NO-BREAK SPACE" (https://www.fileformat.info/info/unicode/char/00a0/index.htm) encoded as UTF-8 (0xc2a0).  Anyway, moving on with life. :)